pom.xml

```diff
@@ -1993,9 +1993,9 @@
         <mongodb.driver.version>4.1.0</mongodb.driver.version>
 
         <!-- Carbon Identity versions -->
-        <carbon.identity.version>5.24.8</carbon.identity.version>
+        <carbon.identity.version>5.25.687</carbon.identity.version>
         <carbon.identity.governance.version>1.7.2</carbon.identity.governance.version>
-        <carbon.identity-inbound-auth-oauth.version>6.9.7</carbon.identity-inbound-auth-oauth.version>
+        <carbon.identity-inbound-auth-oauth.version>6.13.3</carbon.identity-inbound-auth-oauth.version>
```

Comment:
Take a new pull and rebase the changes.identity-inbound-auth-oauth should be 6.13.5